### ISSUE TITLE
OrgSelector not displaying

### ISSUE THREAD

**Issue by bradmac** [Kademi/kademi-dev#5134]
http://viridian-club-reps_viri.kademi.com.au/dashboard

http://viri.admin.kademi.com.au/repositories/reporting-lib/1.2.6/common/theme/apps/reporting/texteditor?fileName=orgSelectorComponent.html

http://viri.admin.kademi.com.au/manageUsers/81939783/#summary-tab

**Comment by bradmac** [comment]
See ReportingApp - listUserOrgs

**Comment by bradmac** [comment]
```
#if ($page)
    #if ($user)
        #set ($isShow = true)
        #set($beans = $applications.Reporting.listUserOrgs($numOrgsInt, $rootFolder.orgData, true, "sales-rep"))
```

**Comment by tuzzmaniandevil** [comment]
Fixed in reporting-lib 1.2.7
Updated KReporting to 1.0.3

**Comment by Uafinder** [comment]
ok, orgselector appears, but it seems have no influence on data displaying on the dashboard
![image](https://user-images.githubusercontent.com/1945230/37716231-ab3b2746-2d26-11e8-8500-369cf7d5841c.png)

**Comment by gonzalon** [comment]
Fixed the console errors in reporting-lib 1.2.8

**Comment by bradmac** [comment]
Seems ok now
